### PR TITLE
D3ASIM-3292: Also allowing None values for initial_soc in the storage…

### PR DESCRIPTION
**Pull request by hannesdiedrich**
… representation.

### CODE REVIEW

**Review by Hassan754 — APPROVED** [review]

File: d3a_interface/schemas.py [hunk 1]
```diff
@@ -79,7 +79,7 @@ class ScenarioSchemas:
                     "number_of_clones": {"type": "number"},
                     "uuid": {"type": "string"},
                     "libraryUUID": {"anyOf": [{"type": "string"}, {"type": "null"}]},
-                    "initial_soc": {"type": "number"},
+                    "initial_soc": {"anyOf": [{"type": "string"}, {"type": "null"}]},
```
Comment (spyrostz) [inline d3a_interface/schemas.py:82]: I am a bit surprised that the soc is not sent as a float but as a string. As an extra precaution and in order to protect ourselves from changes in the future, I would recommend to also allow the number type.

**Review by spyrostz — DISMISSED** [review]
One comment, LGTM once resolved

File: d3a_interface/schemas.py (hunk 1 above)
Comment (hannesdiedrich) [reply d3a_interface/schemas.py]: I actually replaces the string with number, Dont know why this was there.
Also tried it locally, works.

**Review by spyrostz — APPROVED** [review]
LGTM

**Review by Hassan754 — APPROVED** [review]
LGTM

**Pull request merged**